main.go

```diff
@@ -368,6 +369,11 @@ func parseArgs(args []string) {
 	stop := false
 	missing := false
 	noFileFlag := false
+	if v, ok := os.LookupEnv("JOKER_CLASSPATH"); ok {
```

Comment:
I'd really like to remove `JOKER_CLASSPATH` from this initial implementation. I feel like it's here mostly because it's "cute" rather than because it's serving any particular purpose.

Reply:
It indeed has a purpose (avoids need to create a wrapper that serves as a shell, passing `--classpath` in to the Joker executable, for e.g. when ssh'ing in to run a script) but not yet one I actually need, though I anticipate needing it fairly soon.

There are some interesting online discussions about `$CLASSPATH` that raise issues about it, though they pertain to Clojure/Java, which is not intended to serve as a shell (or offer the quick startup that we're after with Joker) -- and, of course, Joker doesn't need a "build tool" to run its programs. E.g.:

https://stackoverflow.com/questions/3639747/environment-variable-classpath-clojures-classpath-why
https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_73/rzaha/classpth.htm

By contrast, Planck (ClojureScript) has its own `$PLANCK_CLASSPATH`, though there's not much discussion as to why or the pros and cons of using it:

http://planck-repl.org/dependencies.html
http://blog.fikesfarm.com/posts/2017-01-22-planck-road-ahead.html

Let me know whether you'd like me to remove this to a (potential) future PR where we can discuss this further. It's not much extra work, and I don't mind scoping down this PR for now.

Reply:
Yep, let's remove this for now.